### ISSUE TITLE
Newer versions of perl have trouble with require 'eds-methods.pl' with strict on

### ISSUE THREAD

**Issue by wizzyrea** [ebsco/edsapi-koha-plugin#46]
The result is that eds-methods.pl won't load, as . isn't in @inc.

**Comment by wizzyrea** [comment]
to replicate, you'll probably need Debian 9

**Comment by Alex-Buckley** [comment]
Hi Liz, 

We found this as well on a Debian 9 machine and running Koha 18.11.x branch. We have found adding ./ to the start of the filename in the require line in  eds-detail.pl, eds-ac.pl, eds-cite.pl, eds-raw.pl, eds-search.pl, pfi-search.pl fixes this issue so I have submitted a pull request with these fixes in it:

https://github.com/ebsco/edsapi-koha-plugin/pull/67

We also found that the line: $self->go_home(); which was removed in the Koha EDS master branch is required otherwise a 500 error is shown upon saving the Koha EDS configuration page. 

Now with this aforementioned fix after saving the config form you are redirected back to the Tools > Tool plugins page in the Koha intranet. 

Cheers,
Alex
